app/src/main/java/com/infomaniak/mail/utils/LocalStorageUtils.kt

```diff
@@ -104,20 +120,36 @@ object LocalStorageUtils {
         }
     }
 
-    fun deleteAttachmentsUploadsDirIfEmpty(
+    fun deleteAttachmentDirIfEmpty(
```

Comment:
Not a big fan of `methodDoSomethingIfSomething` method name. Maybe we can find something else?